java/registry/src/main/java/dev/sunbirdrc/registry/controller/FileStorageController.java

```diff
@@ -40,12 +45,31 @@ public ResponseEntity<DocumentsResponse> save(@RequestParam MultipartFile[] file
         return new ResponseEntity<>(documentsResponse, HttpStatus.OK);
     }
 
-    @DeleteMapping("/api/v1/{entity}/{entityId}/{property}/documents")
-    public ResponseEntity<DocumentsResponse> deleteMultipleFiles(@PathVariable String entity,
+    @PutMapping("/api/v1/{entity}/{entityId}/{property}/documents/{documentId}")//response content type will depend on the default content type configured by the server or the client.
+    public ResponseEntity<DocumentsResponse> update(@RequestParam MultipartFile[] files,
+                                                   @PathVariable String entity,
                                                     @PathVariable String entityId,
-                                                    @PathVariable String property,
-                                                    @RequestBody List<String> files,
-                                                    HttpServletRequest httpServletRequest) {
+                                                   @PathVariable String documentId,
+                                                   @PathVariable String property,
+                                                    HttpServletRequest httpServletRequest) throws ServerException, InsufficientDataException, ErrorResponseException, IOException, NoSuchAlgorithmException, InvalidKeyException, InvalidResponseException, XmlParserException, InternalException//MultipartFile[] represents the updated file(s) that will be uploaded to the server.
+    //httpServletRequest extract information about the incoming request
+    {
+        try {
+            registryHelper.authorize(entity, entityId, httpServletRequest);// authorize() to authorize the request
```

Comment:
remove all comments like this,
https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/